portfolio/src/main/webapp/script.js

```diff
@@ -16,16 +16,18 @@ $(document).ready(function() {
     $('#unfilter-button').css("display","none");
     handleFilterButton();
     handleUnfilterButton();
-  
     $('#2048-pic').css("display","none");
     $('#tweeter-pic').css("display","none");
     $('#job-pic').css("display","none");
     $('#mmr-pic').css("display","none");
     handleClickProject();
-
+    // Enable comment function in main page
     handleQuantityButton();
     handleDeleteButton();
     getComments();
+    // Enable photo uplaod function in photo page
+    fetchBlobstoreUrlAndShowForm();
+    fetchPhotoFormHandler();
```

Comment:
Overall, I think this `script.js` file is getting a bit long. Since you have logic that is both for your homepage and for the photo page in here, I think it would be helpful to migrate all of your photo page logic (filter/unfilter, blobstore stuff) to a separate `photoScript.js` that could be used only on the photo page.